core/app/modules/kit-library/assets/js/components/item-header.js

```diff
@@ -92,11 +92,12 @@ export default function ItemHeader( props ) {
 			{
 				error && <Dialog
 					title={ error }
+					text={ __( 'Nothing to worry about, just try again. If the problem continues, head over to the Help Center.', 'elementor' ) }
 					approveButtonText={ __( 'Learn More', 'elementor-pro' ) }
```

Comment:
```suggestion
					approveButtonText={ __( 'Learn More', 'elementor' ) }
```

Yeah yeah, I know it's an old code...

Reply:
Done